client.go

```diff
@@ -1676,10 +1675,10 @@ func (c *Client) ServerStatus() (*shared.ServerState, error) {
 
 	// Fill in certificate fingerprint if not provided
 	if ss.Environment.CertificateFingerprint == "" && ss.Environment.Certificate != "" {
-		pemCertificate, _ := pem.Decode([]byte(ss.Environment.Certificate))
-		if pemCertificate != nil {
-			digest := sha256.Sum256(pemCertificate.Bytes)
-			ss.Environment.CertificateFingerprint = fmt.Sprintf("%x", digest)
+		var err error
+		ss.Environment.CertificateFingerprint, err = shared.CertFingerprint(ss.Environment.Certificate)
```

Comment:
Same should probably be done to the other two functions that hash certificates (lxd/certificates.go and lxc/remote.go).